src/client/components/UserPage/CreateUrlModal/CreateLinkForm.jsx

```diff
@@ -53,83 +80,207 @@ export default function CreateLinkForm({
+              </Typography>
+            </Button>
+            <Button
+              variant={isFile ? 'contained' : 'text'}
+              className={`${classes.linkTypeButton} ${
+                isFile ? classes.linkTypeButtonEnabled : ''
+              }`}
+              onClick={() => setIsFile(true)}
+            >
+              <FileIcon color={isFile ? '#f9f9f9' : '#384a51'} />
+              <Typography
+                variant="body2"
+                className={classes.linkTypeFileButtonText}
+              >
+                From file
+              </Typography>
+            </Button>
+          </div>
+          {!isFile && (
+            <>
+              <Typography className={classes.labelText} variant="body1">
+                Original link
+              </Typography>
+              <TextField
+                error={!isValidLongUrl(longUrl, true)}
+                InputProps={{
+                  className: classes.outlinedInput,
+                  classes: {
+                    input: classes.input,
+                  },
+                  startAdornment: (
+                    <FormStartAdorment>https://</FormStartAdorment>
+                  ),
+                }}
+                required
+                variant="outlined"
+                placeholder="Enter your link"
+                onChange={(event) => setLongUrl(event.target.value)}
+                value={longUrl}
+                helperText={
+                  isValidLongUrl(longUrl, true)
+                    ? ''
+                    : "This doesn't look like a valid URL."
+                }
+              />
+            </>
+          )}
+          {isFile && (
+            <>
+              <div className={classes.fileInputWrapper}>
+                <Typography className={classes.labelText} variant="body1">
+                  Choose your file
+                </Typography>
+                <div className={classes.maxSizeTextWrapper}>
+                  <InfoIcon />
+                  <Typography variant="caption" className={classes.maxSizeText}>
+                    Maximum size 10mb
+                  </Typography>
+                </div>
+              </div>
+              <div className={classes.fileInput}>
+                <Typography
+                  variant="body2"
+                  className={`${classes.fileNameText} ${
+                    file ? '' : classes.fileNameEmpty
+                  }`}
+                >
+                  {file ? file.name : noFileText}
+                </Typography>
+                <input
+                  type="file"
+                  id="file"
+                  className={classes.fileInputInvis}
+                  ref={fileInputEl}
+                  onChange={(event) => {
+                    const chosenFile = event.target.files[0]
+                    if (!chosenFile) {
+                      return
+                    }
+                    if (chosenFile.size > 10 * 1024 * 1024) {
```

Comment:
This number appears in the codebase multiple times. Instead of having the magic number, could this (and all other instances) turn into a const variable?